app/main.py

```diff
@@ -1,8 +1,26 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: str) -> None:
+        self.name = name
+        self.age = age
+        self.people.update({self.name: self})
```

Comment:
Call class attribute from class, not from an instance
```suggestion
        Person.people.update({self.name: self})
```